.gitlab-ci.yml

```diff
@@ -391,12 +398,14 @@ init_workspace:
     # basic tools
     - apt-get update -q
     - apt-get install -qqy
-      git wget gnupg2 pass autoconf automake build-essential diffstat gawk chrpath
-      libsdl1.2-dev e2tools nfs-client s3cmd psmisc screen libssl-dev python-dev bash
+      git docker.io wget gnupg2 pass autoconf automake build-essential diffstat gawk chrpath
+      libsdl1.2-dev e2tools nfs-client s3cmd psmisc screen libssl-dev python-dev bash curl
       libxml2-dev libxslt-dev libffi-dev nodejs libyaml-dev sysbench texinfo pkg-config
       zlib1g-dev libaio-dev libbluetooth-dev libbrlapi-dev libbz2-dev libglib2.0-dev
       libfdt-dev libpixman-1-dev zlib1g-dev jq liblzo2-dev device-tree-compiler
       qemu-system-x86 qemu-system-arm bc kpartx liblzma-dev cpio sudo awscli gdisk
+    # Verify correct dind setup
+    - docker ps
```

Comment:
Why `ps` instead of `version`? I think `version` is just more informative and used already in other jobs.

Reply:
I initially thought that `docker version` didn't use the API, so that's why I changed it. Turns out it does use the API to get the server parameters, so I'll revert this one back to `docker version`.